pages/instructorAssessmentInstances/instructorAssessmentInstances.ejs

```diff
@@ -104,6 +108,7 @@
                 <th class="text-center">Role</th>
                 <th class="text-center">Instance</th>
                 <th class="text-center">Score</th>
+                <th class="text-center">Date Started</th>
```

Comment:
```suggestion
                <th class="text-center">Date started</th>
```

We prefer sentence case (leading capital letter) for column headings and table descriptions.